### ISSUE TITLE
idea for event: Glint in the mine

### ISSUE THREAD

**Issue by AnastaseAlex** [TechpriestEnginseer/Community-Patch-Events-Development#3]
Glint in the mine
Requires: Mine without resources, iron revealed, ancient era.

text: a worker in the mines of [cityname] saw a glint in the darkness. The miners are too afraid to continue working there.

case1: Send the army to investigate
Requires a garrisoned unit in the city, garrison unit gains +15 xp
Text: The army found a band of bandits hiding in the mines and drove them out.

case2: Send a shaman
Requires a shrine. You get 30 faith. If earth mother is the pantheon mine gets +2 faith.
Text: The priest tamed the spirits that laid there and gave us access to their wisdom.

case3: Send a wise man
Requires a council. The mine gets 1 iron.

case4: Send a merchant
Requires market. The mine gets +2 gold
This semi-precious material could actually be sold quite profitably.

case5: Abandon the mine
The mine is destroyed.

**Comment by TechpriestEnginseer** [comment]
Sounds great, you should however revise case 3 since we cannot do separate percentages without using Lua. Something I could do, but we're trying to avoid here.

Otherwise, it looks good. I'll start working on it.

**Comment by AnastaseAlex** [comment]
I updated it.

**Comment by AnastaseAlex** [comment]
What are the needs at the moment? It would be nice to have a readme with how many positive/negative needs we have for events.

**Comment by Hokath** [comment]
By needs, do you mean requirements, like garrisoned units?
They are listed in (1) Community Patch\Core Files\Core Tables\GameEventsTable.xml

**Comment by Hokath** [comment]
For the Choice 2, currently RequiresPantheon is boolean. I don't think we have a handle for religious beliefs at the moment, could ask for one but I don't know if it makes too much of a difference thematically.

**Comment by TechpriestEnginseer** [comment]
You could however use Lua to handle the religious beliefs, but I don't want to get into those since Lua religion is difficult to test out.

**Comment by Hokath** [comment]
Have you started this one, Techpriest? 
If not I could do it, and you could do the other one - looks too lua heavy for me

**Comment by TechpriestEnginseer** [comment]
You can do this one.

**Comment by AnastaseAlex** [comment]
@Hokath no more about what type of event ideas do I need to come up with? Ancient era? Renaissance? Positive? Negative?

**Comment by AnastaseAlex** [comment]
Also we can just change case2 to if you have pantheon, mine gets +1 faith, else get 30 faith.

**Comment by Hokath** [comment]
Yeah we can just have two different choices. One that needs a pantheon and one that doesnt

Sent from Yahoo Mail on Android 

  On Sat, 18 Jun, 2016 at 18:28, AnastaseAlexnotifications@github.com wrote:  
Also we can just change case2 to if you have pantheon, mine gets +1 faith, else get 30 faith.

**Comment by Hokath** [comment]
> no more about what type of event ideas do I need to come up with? Ancient era? Renaissance? Positive? Negative?

Oh well this is a bit subjective I guess. Personally I think more ancient and classical city events, since you always get that monument a bunch at the start and we could do with mixing it up

**Comment by Hokath** [comment]
Ok, you can see it in the code tab
